cypress.json

```diff
@@ -0,0 +1,4 @@
+{
+  "baseUrl": "http://localhost:1234",
+  "chromeWebSecurity": false
```

Comment:
Just curious what this does?

Reply:
I think this bypasses the **cross-origin** errors and lets you load your page in an iframe.

We don't need it tho. It must be a `debugging` code I wrote and forgot to delete. I will remove it.